bin/civi-test-run

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -e
+
+if [ -z $1 ]; then
+  echo "No Buildkit path passed in"
```

Comment:
This part can be determined automatically. Copy/paste the preamble from one of the other bash scripts, eg

https://github.com/civicrm/civicrm-buildkit/blob/1a9922da3bec95b7b9d5bb9731f31ce193a8b756/bin/civi-download-tools#L8-L17

(`PRJDIR` in `civi-download-tools` is the same as `BLDKIT` in `civi-test-run`.)